### PR TITLE
Feat/user

### LINKED ISSUE #1: time to improve this blog don't you think ?

### LINKED ISSUE #3: outDir for tsconfig.json
You have both .js and .ts file on the same folder. This needs to be avoided.
Check the _output directory_ option in tsconfig to fix this

### PR DESCRIPTION
**Pull request by bcdbuddy**

### CODE REVIEW

**Comment by kouna77** [comment]
Yes it's true, just that when I ran the project, I had errors and after having corrected them, I wanted to compile again the index.ts and MongoHepler.ts files and that led me to errors again and also the compiled files instead of being on dist is found in src, I don't understand why because it is well specified in the tsconfig file, I just did tsc index.ts the same thing MongoHelper.ts

**Comment by bcdbuddy** [comment]
> Yes it's true, just that when I ran the project, I had errors and after having corrected them, I wanted to compile again the index.ts and MongoHepler.ts files and that led me to errors again and also the compiled files instead of being on dist is found in src, I don't understand why because it is well specified in the tsconfig file, I just did tsc index.ts the same thing MongoHelper.ts

just run `tsc --watch` without any file it's good. remove them before moving on. That's way more important than the actual coding
Fix that first

**Comment by kouna77** [comment]
@bcdbuddy je viens de faire un push

**Comment by bcdbuddy** [comment]
Tkt Je vois tous tes pushes fais un PR quand c'est bon

Cordialement,
BCD
Babacar Cissé Dia
Senior développeur Web, mobile & Desktop
Ingénieur gestion de données et ingénierie logicielle
https://babacar-cisse-dia.com
________________________________

File: routes/comment.js [hunk 1]
```diff
@@ -12,7 +12,7 @@ const express=require('express')
 const router=express.Router()
 const TYPE_ERROR = 'error'
 const TYPE_SUCCESS = 'success'
-const verifyToken = require('./verify_token')
+const verifyToken = require('../src/routes/verify_token')
```
Comment (bcdbuddy) [inline routes/comment.js:15]:
```suggestion
const verifyToken = require('../src/middlewares/verify_token')
```

**Pull request merged**